### PR TITLE
Implement getting a single article

### PR DESCRIPTION
**Pull request by JohnDMurphy**
-Get a single article
-Minor styling change and addition for new page

### CODE REVIEW

File: src/components/SingleArticle.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      setArticle(articleData);
+    });
+  }, []);
+
+  return (
+    <>
+      <div className='single-article'>
+        <Link to={`/`}>
+          <h3 className='article-header'>{article.title}</h3>
+        </Link>
```
Comment (DominicFH) [inline src/components/SingleArticle.jsx]: From a user point of view, I'm not sure if this should be a link. If we're navigating directly to view a specific article then we may just want this to be the `<h3>` itself. Clicking on an article title to navigate to the home screen to view all titles doesn't seem intuitive :)

File: src/components/SingleArticle.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <p>{article.body}</p>
+        <p className='author-text'>
+          <small>
+            By <br />
+            {article.author}
+          </small>
+        </p>
+        <Link to={`/`}>
+          <h6 className='article-header back-btn'>Go Back</h6>
+        </Link>
```
Comment (DominicFH) [inline src/components/SingleArticle.jsx:29]: I love the idea of having a Go Back button but I'd maybe make it a bit more descriptive. Go Back suggests going to the previous page but if you go to the articles by topic and then view a specific article, I'd assume a Go Back button to take me back to the articles by topic but it takes you to all articles (This is me being very nitpicky though!!)

**Review by DominicFH — COMMENTED** [review]
Nice stuff! :) I like how you've got some foundations for future tickets and seemed to have had a plan about how to approach the app :D

File: src/components/SingleArticle.jsx (hunk 1 above)
Comment (JohnDMurphy) [reply src/components/SingleArticle.jsx]: This is no longer a link

File: src/components/SingleArticle.jsx (hunk 2 above)
Comment (JohnDMurphy) [reply src/components/SingleArticle.jsx:29]: Changed to Home Page so it is more intuitive i think

**Review by DominicFH — COMMENTED** [review]
Nice! Yeah Home Page is a great descriptor for it I'd say, merge away :D

**Pull request merged**